src/foam/comics/v2/userFeedback/UserFeedbackException.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.comics.v2.userFeedback',
+  name: 'UserFeedbackException',
```

Comment:
add documentation:, to this model and others

Reply:
Done.